src/lib/modules/console/suggestions.js

```diff
@@ -60,7 +61,19 @@ class Suggestions {
     suggestions.push({value: 'web3', command_type: "javascript object", description: "instantiated web3.js object configured to the current environment"});
     suggestions.push({value: 'EmbarkJS', command_type: "javascript object", description: "EmbarkJS static functions for Storage, Messages, Names, etc."});
 
-    return cb(results);
+    if (cmd[cmd.length - 1] === '.') {
```

Comment:
That's cool, but we could make it even better. Where as soon as there is a `.`, we run it the same way you did, but we strip the unfinished word and do a match on the result with the unfinished word. Eg:
I type: `Web3.eth.getAcc`
The function will strip `.getAcc` and run the eval. It will return all the functions of `Web3.eth` and then we loop those and return only the functions that match `getAcc` => `getAccounts`
(I know we already have support for `getAccounts`, just wanted an easy example)